packages/api-key-service/src/functions/tests/mock.hander.context.ts

```diff
@@ -0,0 +1,20 @@
+import { APIGatewayEventRequestContextWithAuthorizer } from 'aws-lambda'
+
+export const mockRequestHandlerContext = (
+  body: { [s: string]: any },
```

Comment:
See my comment about a StringMap type on the other PR - no big deal but just a suggestion